src/constants/commons.ts

```diff
@@ -0,0 +1,3 @@
+export const MISSED_UPDATES_PROD_ROLE_ID = '1183553844811153458';
+export const MISSED_UPDATES_STAGING_ROLE_ID = '1183553844811153458';
+export const MISSED_UPDATES_DEVELOPMENT_ROLE_ID = '1181214205081296896';
```

Comment:
should this be in config instead of constants ?

Reply:
Do you mean as environment variable?

Reply:
Yeah right

Reply:
This is not a secret. 
- Anyone within our discord server can access it.
- Anyone outside have no use for it

Reply:
Its also done the same way in our backend repo:
https://github.com/Real-Dev-Squad/website-backend/blob/develop/config/production.js